index.js

```diff
@@ -6,10 +6,14 @@ const wrapAnsi = require('wrap-ansi');
 const main = stream => {
 	let prevLineCount = 0;
 
+	const getWidth = function (columns) {
+		return (columns && columns > 0) ? columns - 1 : 80;
+	};
+
 	const render = function () {
 		cliCursor.hide();
 		let out = [].join.call(arguments, ' ') + '\n';
-		out = wrapAnsi(out, process.stdout.columns || 80, {wordWrap: false});
+		out = wrapAnsi(out, getWidth(process.stdout.columns), {hard: true, wordWrap: false});
```

Comment:
Could you use https://github.com/sindresorhus/term-size instead of `process.stdout.columns` too?